.circleci/config.yml

```diff
@@ -353,47 +355,78 @@ workflows:
       - linux_oqs:
           <<: *require_buildcheck
           name: ubuntu-focal-shared-noopenssl
           context: openquantumsafe
           CONTAINER: openquantumsafe/ci-ubuntu-focal-x86_64:latest
           CMAKE_ARGS: -DCMAKE_C_COMPILER=gcc-7 -DOQS_DIST_BUILD=OFF -DOQS_USE_OPENSSL=OFF -DBUILD_SHARED_LIBS=ON
-          PYTEST_ARGS: --ignore=tests/test_namespace.py --ignore=tests/test_leaks.py --numprocesses=auto
+          PYTEST_ARGS: --ignore=tests/test_namespace.py --ignore=tests/test_leaks.py --ignore=tests/test_kat_all.py --numprocesses=auto
       - linux_oqs:
           <<: *require_buildcheck
           name: ubuntu-focal-clang15
           context: openquantumsafe
           CONTAINER: openquantumsafe/ci-ubuntu-focal-x86_64:latest
           CMAKE_ARGS: -DOQS_STRICT_WARNINGS=ON -DCMAKE_C_COMPILER=clang-15 -DOQS_OPT_TARGET=skylake
+          PYTEST_ARGS: --ignore=tests/test_kat_all.py
       - linux_oqs:
           <<: *require_buildcheck
           name: ubuntu-bionic-i386
           context: openquantumsafe
           CONTAINER: openquantumsafe/ci-ubuntu-bionic-i386:latest
           CMAKE_ARGS: -DCMAKE_TOOLCHAIN_FILE=../.CMake/toolchain_x86.cmake
-          PYTEST_ARGS: --ignore=tests/test_leaks.py
+          PYTEST_ARGS: --ignore=tests/test_leaks.py --ignore=tests/test_kat_all.py
       - arm_machine:
           <<: *require_buildcheck
           name: arm64
-          PYTEST_ARGS: --numprocesses=auto --maxprocesses=10
+          PYTEST_ARGS: --numprocesses=auto --maxprocesses=10 --ignore=tests/test_kat_all.py
       - macOS:
           <<: *require_buildcheck
           name: macOS-gcc11
           CMAKE_ARGS: -DCMAKE_C_COMPILER=gcc-11
+          PYTEST_ARGS: --ignore=tests/test_kat_all.py
       - macOS:
           <<: *require_buildcheck
           name: macOS-noopenssl
           CMAKE_ARGS: -DOQS_STRICT_WARNINGS=ON -DOQS_USE_OPENSSL=OFF
+          PYTEST_ARGS: --ignore=tests/test_kat_all.py
       - macOS:
           <<: *require_buildcheck
           name: macOS-shared
           CMAKE_ARGS: -DBUILD_SHARED_LIBS=ON -DOQS_DIST_BUILD=OFF -DOQS_ENABLE_KEM_CLASSIC_MCELIECE=OFF
+          PYTEST_ARGS: --ignore=tests/test_kat_all.py
 
   commit-to-main:
     when:
-      equal: [ main, << pipeline.git.branch >> ]
+      and:
+        - not:
+            equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+        - equal: [ main, << pipeline.git.branch >> ]
     jobs:
       - trigger-downstream-ci:
           context: openquantumsafe
+
+  nistkat:
+    when:
+      and:
+        - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+        - equal: [ "NIST KATs", << pipeline.schedule.name >> ]
+        - equal: [ sw-full-kat, << pipeline.git.branch >> ]
+    jobs:
+      - linux_oqs:
+          name: nistkat-generic
+          context: openquantumsafe
+          CONTAINER: openquantumsafe/ci-ubuntu-focal-x86_64:latest
+          CMAKE_ARGS: -DOQS_OPT_TARGET=generic
```

Comment:
This arg alone shouldn't work as OQS_DIST_BUILD=ON by default: See https://github.com/open-quantum-safe/liboqs/blob/main/CONFIGURE.md#oqs_opt_target

Reply:
Good catch, thanks! I'll change it to set `-DOQS_DIST_BUILD=OFF`.